### ISSUE TITLE
Attempts not being adding to count for Life Binder's/Invincible

### ISSUE THREAD

**Issue by Cybrosaen** [SacredDuckwhale/Rarity#421]
Whenever I do a new weekly attempt for Invincible (271/100) and Life Binder's Handmaiden (260/100), the stat counter does not update with the newest attempt and chat box line also doesn't have note to say for example "Invincible: 271 attempts". I have had to manually update these now for the past few weeks. They are being tracked currently and all the settings seem to be fine, so I'm not sure why just these 2 recently refuse to automatically count and update.

**Comment by Cybrosaen** [comment]
Another update: I tried Deathwing on my alt that had never done it before and the count went up in rarity/achievement ui also, so the issue for these particular encounters are definitely linked to my main character it appears.

**Comment by Duckwhale** [comment]
Yeah, that is exactly my point: Rarity takes the statistics from Blizzard, and if they don't update it won't detect the attempt either.

I've no idea why they break the statistics every once in a while, presumably a lot of spaghetti code on Blizzard's end? Usually they stop working when they introduce Timewalking for a raid, but that's no guarantee it won't break in other scenarios, as well.

If only one character is affected I'd first make sure the difficulty is set correctly, ingame statistics aren't increasing either, and once you're sure it's actually broken on Blizzard's end report it as a bug (ingame, Help section of the Escape menu).

Then, all you have to do is wait a few years, or decades, and maybe they will get around to fixing it eventually :S
